### DISCUSSION TITLE
Allow npm scripts to be read as a array

### DISCUSSION THREAD

**Discussion by Arcanorum** [npm/rfcs#661] [RRFC]
Basically this: https://github.com/npm/npm/issues/6943

> Sometimes when writing npm scripts, they can start to get a little bit unwieldy if you try to string multiple commands together. Take for example this:
>
>```
>"scripts": {
>  "build": "npm run build:scripts -s && npm run build:styles -s && npm run build:markup -s",
>  "build:scripts": "browserify -d assets/scripts/main.js -p [minifyify --compressPath . --map main.js.map --output dist/main.js.map] | hashmark -n dist/main.js -s -l 8 -m assets.json 'dist/{name}{hash}{ext}'"
>}
>```
>
> It'd be real swell if npm could read a scripts value as an array, and just run .join('') before executing, which in my humble opinion would improve the readability of scripts immensely:
> 
> ```
>"scripts": {
>   "build": [
>     "npm run build:scripts -s",
>     "&& npm run build:styles -s",
>     "&& npm run build:markup -s"
>   ],
>   "build:scripts": [
>     "browserify -d assets/scripts/main.js -p ",
>     "[minifyify --compressPath . --map main.js.map --output dist/main.js.map]",
>     " | hashmark -n dist/main.js -s -l 8 -m assets.json 'dist/{name}{hash}{ext}'"
>   ]
> }
>```

There was huge support for this but got repeatedly shut down for no valid reason.

Would be big QOL improvement.

**Comment by olleharstedt** [comment]
Any progress here? Silly it's still missing...

Reported 2014 already: https://github.com/npm/npm/issues/6943
